ee/tabby-webserver/src/service/background_job/git.rs

```diff
@@ -52,10 +52,6 @@ impl SchedulerGitJob {
             .map(|repo| CodeRepository::new(&repo.git_url, &repo.source_id))
             .collect();
 
-        let mut code = CodeIndexer::default();
-
-        code.garbage_collection(&repositories).await;
```

Comment:
only contains repositories from git provider, thus the cloned dirs from github / gitlab will be wrongly deleted.

@xxs-wallace @liangfung

Reply:
Can we add repository from github/gitlab to this variable `repositories` ?

Reply:
> Can we add repository from github/gitlab to this variable repositories ?

it's done in IndexGarbageCollection now